src/Raku/ast/variable-declaration.rakumod

```diff
@@ -649,15 +649,20 @@ class RakuAST::VarDeclaration::Simple
                                 target => RakuAST::ParameterTarget::Var.new('$_')
                             )
                         ])
-                    )),
+                        )),
```

Comment:
Extra indent?

Reply:
That's Comma behaving badly. Will fix, thanks!